core/site.go

```diff
@@ -469,6 +483,41 @@ func (site *Site) updatePvMeters() {
 	site.publish(keys.Pv, mm)
 }
 
+// updateLogMeters updates log meters. All measurements are optional.
+func (site *Site) updateLogMeters() {
+	if len(site.logMeters) == 0 {
+		return
+	}
+
+	mm := make([]meterMeasurement, len(site.logMeters))
+
+	for i, meter := range site.logMeters {
+		// log power
+		power, err := backoff.RetryWithData(meter.CurrentPower, bo())
+		if err == nil {
+		} else {
+			site.log.ERROR.Printf("logMeter %d power: %v", i+1, err)
```

Comment:
```suggestion
			site.log.ERROR.Printf("log meter %d power: %v", i+1, err)
```

Reply:
changed in [f13574e](https://github.com/evcc-io/evcc/pull/15049/commits/f13574e0f1934d51499798400de4270fe815cef4)